### DISCUSSION TITLE
Can we modify the channels in the m3u file, by .txt?

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#183] [Q&A] [answered]
Hi,

I would like to please know, if I add more channels to this file: https://iptv-org.github.io/iptv/index.m3u
by downloading the m3u file and opening the file via .txt, then adding more lines of channels I know work and then save the file as m3u again, **will I still receive the updates** to the channels you perform or will I no longer be able to receive the updates to this file from your site, once I add new lines of channels to this new m3u file?

Thanks so much!

**Comment by what-a** [comment, marked as answer]
No, you will not receive updates. Think of it this way... 
That playlist file resides on a github server where it's updated daily. When you download it, you are making a copy (of how it looked on one particular day) onto your local computer which thereafter only you are able to update. The file on the github server and the file on your local computer will quickly fall out of sync with each other (unless you periodically download a fresh copy).

**Reply by jphilippe2017** [reply]
Great, thanks for your answer. I guess the best way is just to download the original file and request the channels missing that I am looking for then. I appreciate your fast reply!!
